### PR TITLE
Stop all chromium zombie processes when SanePDFReports script finished

### PR DESCRIPTION
**Pull request by dorsha**
## Status
- [ ] In Progress (tests)
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29665
fixes: https://github.com/demisto/etc/issues/29671

## Description
Make sure to stop all chromium zombie processes when SanePDFReports script finished.

## Screenshots
image

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests - in progress
- [x] Documentation - Release Notes

### CODE REVIEW

**Comment by glicht** [comment]
@dorsha can we get  a test playbook for this script? This bug would have been caught by our testing framework if we had a test playbook, as we verify docker processes when a test is done.

**Comment by dorsha** [comment]
I think that we can do this in unit test

**Comment by dorsha** [comment]
Did not forget this one, closing some other stuff and will get back to this.

File: Packs/Base/Scripts/SanePdfReport/SanePdfReport_test.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import demistomock as demisto
+from SanePdfReport import *
+import subprocess
+import os
+import logging
+
+# disable warning from urllib3. these are emitted when python driver can't connect to chrome yet
+logging.getLogger("urllib3").setLevel(logging.ERROR)
```
Comment (Itay4) [inline Packs/Base/Scripts/SanePdfReport/SanePdfReport_test.py:8]: please remove the lines taken from rasterize

**Review by Itay4 — APPROVED** [review]

File: Packs/Base/Scripts/SanePdfReport/SanePdfReport_test.py (hunk 1 above)
Comment (dorsha) [reply Packs/Base/Scripts/SanePdfReport/SanePdfReport_test.py:8]: done

**Review by Itay4 — APPROVED** [review]

**Comment by kirbles19** [comment]
@dorsha Done.

**Pull request merged**